api/ws/workers/peers_update_rules.js

```diff
@@ -67,7 +67,6 @@ PeersUpdateRules.prototype.insert = function(peer, connectionId, cb) {
 			peer,
 			err => {
 				if (err) {
-					connectionsTable.remove(peer.nonce);
```

Comment:
Shouldn't we move also the `connectionsTable.add`, after the operation succeeds on the master process, to line https://github.com/LiskHQ/lisk/blob/6feb9a42e13884453b4de3fc64760acdf3af7ee5/api/ws/workers/peers_update_rules.js#L80?